apps/desktop-wallet/src/api/addresses.ts

```diff
@@ -30,7 +30,9 @@ import {
 
 const PAGE_LIMIT = 100
```

Comment:
Not directly linked to this PR but this could be extracted as well (I actually thought it was already). WDYT?

Reply:
To be discussed later:
- For maintainability sake, we could use comments to describe what each constant does. 
- Some constants would be in SHRD, those ones should IMO be imported and re-declared in each sub-package for clarity, instead of being imported directly in the code.

Reply:
Hm, why should we re-declare them? We can easily click on the constant in our IDEs and navigate to the source where we could have the comments that you mentioned.